prereq_flowchart/TEMP_graphviz/test.py

```diff
@@ -62,7 +68,9 @@ def data_graph(data: List[Dict[str, Union[str, List[str]]]]) -> graphviz.Digraph
 if __name__ == "__main__":
     if not exists(IMAGEDIR):
         mkdir(IMAGEDIR)
-    data_graph(read_data(join(dirname(__file__), "dummy_data.json"))).render(
+    # data_graph(read_data(join(dirname(__file__), "dummy_data.json"))).render(
+    # Computer Science B.S.
+    data_graph(read_data(join("../../data", "4e6dca0766445d116f5b335dcfcad3d4.json"))).render(
```

Comment:
wondering where this hash value comes from?

Reply:
i hash the files so that there aren't any filename issues with whatever characters the majors are using (mostly periods), but this was a test written before i made the search utility, and was meant to be a "backend" thing. you can still manually find a major from "lookup.json" in the data folder